### ISSUE TITLE
The problem for not fixing BN

### ISSUE THREAD

**Issue by powermano** [yangxue0827/FPN_Tensorflow#21]
What is the Map in pascal voc 07 dataset using this code?  As i read your code, i find that you do not fix the BN parameters.  The batchsize is small, following the paper FPN or Mask RCNN, you need to set the training and trainbale to False for the slim.batch_norm op. I am testing whether this is the problem for the  bad generalization for mutiple objects.

**Comment by yangxue0827** [comment]
We have also suspected, but it does not seem to be. Look forward to your results.

**Comment by powermano** [comment]
I have tried the following changes(I only train the network on voc2007 and test on voc2007):
1. Fix BN parameters :   The result does not improve a lot, only 1%(The final result is also too low for training on voc2007 and test on voc2007)
2. RGB and BGR issue.:  As the slim released resnet101 models is trained on the RGB images, i both test both cases. The final result on RGB images(fix BN) is about 62,3%, and the final result on BGR is about 59.9%(fix BN). The sheep, bottle, pottedplant, boat MAP are extremely low(about 30-40%) 
3. Eval : I change your eval code, following the code based on py-faster-rcnn written by rbgirshick. And the above results are all tested on the changed eval code.

I also find some differerces between the original faster-rcnn and your implementation. I will test them.

From the training processing, i find the rpn loc loss for the first 10000 steps is too low which is unusual（most for the images, not all）.  

Did you find any problems with your code?

**Comment by yangxue0827** [comment]
We rewrote the entire project (faster-rcnn) and achieved better results (res50-75%, res101-77%, mobilenetv2-50%). The code will be uploaded to https://github.com/DetectionTeamUCAS. @powermano

**Comment by powermano** [comment]
I have tried the faster-rcnn written by endernewton(tf-faster-rcnn). I have been using this code for my research and project. Training on voc2007, the test result on voc 2007 is 75.7%. 

I will further test some part of the code, hoping to identify the problem.

**Comment by yangxue0827** [comment]
@powermano  https://github.com/DetectionTeamUCAS/Faster-RCNN_Tensorflow

**Comment by powermano** [comment]
Thanks.
